bazel/envoy_internal.bzl

```diff
@@ -7,7 +7,7 @@ def envoy_copts(repository, test = False):
     posix_options = [
         "-Wall",
         "-Wextra",
-        "-Werror",
+#        "-Werror", FIXME: https://issues.redhat.com/browse/OSSM-1201
```

Comment:
We could remove this line?

Reply:
We could, but I slightly prefer to keep those around, as they keep bothering us, thus motivating us to not forget about them.